components/index.ts

```diff
@@ -1,96 +1,393 @@
-  RadiusProps,
-  SizeProps,
-  FillProps,
-  ClassStyleProps,
-  ChildrenProps
-} from './types.js'
-
-export { ModalContext, ModalProvider, useModal } from './element/Modal';
-export {
-  Thead,
-  Tfoot,
-  Tcol,
-  Trow,
-  Tgroup
-} from './element/Table';
-export { useFieldset } from './form/Fieldset.js';
-
-import Alert from './element/Alert';
-import Badge from './element/Badge';
-import Loader from './element/Loader';
-import Modal from './element/Modal';
-import Table from './element/Table';
-import Tooltip from './element/Tooltip';
-import Button from './Button.js';
-import Control from './form/FieldControl.js';
-import Fieldset from './form/Fieldset.js';
+  TabsContextProps,
+  TabsHeadProps,
+  TabsLabelProps,
+  TabsBodyProps,
+  TabsContentProps,
+  TabsActiveProps,
+  TabsInactiveProps,
+  TabsProps
+} from './Tabs.js';
+import Tabs from './Tabs.js';
 
-import setColorClass from './helpers/color/setColorClass.js';
-import setBackgroundColorClass from './helpers/color/setBackgroundColorClass.js';
-import setBorderColorClass from './helpers/color/setBorderColorClass.js';
-import setTextColorClass from './helpers/color/setTextColorClass.js';
-import setTextAlignClass from './helpers/setTextAlignClass.js';
-import setDisplayClass from './helpers/setDisplayClass.js';
-import setRadiusClass from './helpers/setRadiusClass.js';
-import setTextSizeClass from './helpers/setTextSizeClass.js';
+//tooltip
+export type { 
+  TooltipConfig,
+  TooltipProps,
+  TooltipDirection
+} from './Tooltip.js';
+import Tooltip from './Tooltip.js';
 
 export {
-  Element,
-  Field,
-  Form,
-  Format,
+  Accordion,
   Alert,
   Badge,
+  Bread,
+  Button,
+  Dialog,
   Loader,
-  Modal,
+  Notifier,
+  Pager,
+  Progress,
   Table,
-  Tooltip,
-  Button,
-  Control,
+  Tabs,
+  Tooltip
+};
+
+//--------------------------------------------------------------------//
+// Helpers
+
+//tools
+export { BackgroundColorTool } from './helpers/tools/BackgroundColorTool.js';
+export { BorderColorTool } from './helpers/tools/BorderColorTool.js';
+export { BorderRadiusTool } from './helpers/tools/BorderRadiusTool.js';
+export { ColorTool } from './helpers/tools/ColorTool.js';
+export { DisplayTool } from './helpers/tools/DisplayTool.js';
+export { FillTool } from './helpers/tools/FillTool.js';
+export { PropTool } from './helpers/tools/PropTool.js';
+export { TextAlignTool } from './helpers/tools/TextAlignTool.js';
+export { TextColorTool } from './helpers/tools/TextColorTool.js';
+export { TextSizeTool } from './helpers/tools/TextSizeTool.js';
+
+export { getClassStyles } from './helpers/getClassStyles.js';
+export { getSlotStyles } from './helpers/getSlotStyles.js';
+export { removeThemeProps } from './helpers/removeThemeProps.js';
+export { toChildrenArray } from './helpers/toChildrenArray.js';
+
+//--------------------------------------------------------------------//
+// Blocks
+
+//--------------------------------------------------------------------//
+// Form
+
```

Comment:
This entire section is incorrectly done. The `export type` lines are correct but you should have done as we discussed before.

```
import * as Form from './form/index.js';
```

then export it. make sure there is only one `export {...};`  in the file.